lib/auth/webauthncli/fido2_test.go

```diff
@@ -130,7 +130,7 @@ func (p pinCancelPrompt) PromptTouch() {
 }
 
 func TestIsFIDO2Available(t *testing.T) {
-	const fido2Key = "TSH_FIDO2"
+	const fido2Key = "TELEPORT_FIDO2"
 	defer func() {
 		os.Unsetenv(fido2Key)
```

Comment:
Would it makes sense to convert over to `t.Setenv` for these tests so you don't have to unset anything?

Reply:
I concur with using `t.Setenv`. Another option seen elsewhere in `tsh` tests is to supply mock `os.LookupEnv` function to the function being tested. This has the benefit of having the global process state untouched.

Reply:
Done.

I don't think a more evolved mocking solution is worth it, if I'm not mistaken env changes would only affect the webauthncli.test binary.